### PR TITLE
Fix grammar and formatting

### PR DESCRIPTION
**Pull request by glibg10b**
Most grammatical errors were misused capital letters. Formatting was okay, but I made it more markdown-compliant.

`glibg10b#4504` on Discord.

### CODE REVIEW

File: Utilities/debloating.md [hunk 1]
```diff
@@ -3,33 +3,35 @@ title: Debloating Windows
...
 
-**[WindowsSpyBlocker](https://github.com/crazy-max/WindowsSpyBlocker)** - Created to intercept your Network Traffic and block Tracking requests even before they get to Microsoft.
+# Honorable Mentions
 
-**[Fix-Windows-Privacy](https://modzero.github.io/fix-windows-privacy/)** - Deletes Telemetry Registry keys that may have been left over by other Programs.
- 
- 
-# Honorable Mentions 
-[**Sigma-Deleter**](https://github.com/XatzClient/Sigma-Deleter) - Deletes every trace of the Miner placed by the Sigma Minecraft Client.
+[**Sigma-Deleter**](https://github.com/XatzClient/Sigma-Deleter) - Deletes every trace of the miner placed by the Sigma Minecraft client.
```
Comment (doggo404) [inline Utilities/debloating.md:31]: leave this in.

File: Utilities/misc.md [hunk 2]
```diff
@@ -5,127 +5,125 @@ description: Tools and Apps to make your life easier.
...
+[**Cookie AutoDelete**](https://github.com/Cookie-AutoDelete/Cookie-AutoDelete) - Deletes cookies and other browsing site data as soon as a browser tab closes, domain changes or browser restarts.
 
-[**Disconnect**](https://github.com/disconnectme/disconnect) - Visualize and block the invisible websites that track your Search and browsing history.
+[**Disconnect**](https://github.com/disconnectme/disconnect) - Visualize and block the invisible websites that track your search and browsing history.
 
-[**ViolentMonkey**](https://github.com/violentmonkey/violentmonkey) | [**Tampermonkey**](https://www.tampermonkey.net/) - Browser UserScript Managers.  
+[**ViolentMonkey**](https://github.com/violentmonkey/violentmonkey) | [**Tampermonkey**](https://www.tampermonkey.net/) - Browser userscript managers.  
 
 # Browser UserScripts
-<sub>*You need a Browser UserScript Manager.*</sub>
```
Comment (doggo404) [inline Utilities/misc.md:58]: leave this in.

File: Utilities/misc.md [hunk 3]
```diff
@@ -5,127 +5,125 @@ description: Tools and Apps to make your life easier.
...
 
 [**Spyware Neocities**](https://spyware.neocities.org/articles/) - Site which periodically analyzes apps for spying behavior and helps you mitigate it.
 
-[**File Hosting Wiki**](https://filehostlist.miraheze.org/wiki/Main_Page) - Wiki for Premium Leechers, Torrent Leechers and Debrids.
-
-[**YtmDL Web**](https://ytmdl.deepjyoti30.dev/) - Best looking and performing Online YouTube Music Downloader, no intrusive Ads.
-
-[**Outline**](https://outline.com/) - Removes clutter from News Articles.
-
-[**Steam Workshop Downloader**](https://www.steamworkshopdownloader.io/) - Download Steam Workshop Items without having to own the game.
```
Comment (doggo404) [inline Utilities/misc.md:120]: leave all of these in

**Comment by rippedpiracy** [comment]
I've created the https://github.com/rippedpiracy/docs/tree/dev branch where we will continue to review this.

**Pull request closed without merging**